src/Microsoft.ComponentDetection.Detectors/pip/PipReportComponentDetector.cs

```diff
@@ -294,8 +294,15 @@ protected override async Task OnFileFoundAsync(ProcessRequest processRequest, ID
             // if pipreport fails, try to at least list the dependencies that are found in the source files
             if (this.GetPipReportOverrideBehavior() != PipReportOverrideBehavior.SourceCodeScan && !this.PipReportSkipFallbackOnFailure())
             {
-                this.Logger.LogInformation("PipReport: Trying to Manually compile dependency list for '{File}' without reaching out to a remote feed.", file.Location);
-                await this.RegisterExplicitComponentsInFileAsync(singleFileComponentRecorder, file.Location, pythonExePath);
+                try
+                {
+                    this.Logger.LogInformation("PipReport: Trying to Manually compile dependency list for '{File}' without reaching out to a remote feed.", file.Location);
```

Comment:
we should make a note here and in the docs that if users see this message, python detection essentially isn't happening

Reply:
made the note here, and it is already in the pip report TSG doc. Our wrapper makes it so that we do not fallback to this logic, so internal users should not hit this